pyproject.toml

```diff
@@ -5,7 +5,7 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.poetry]
 name = "dominodatalab-data"
-version = "0.4.1dev2"
+version = "0.4.1dev3"
```

Comment:
you shouldn't be changing this unless you've published something to pypi

Reply:
definitely don't change this you should be rebasing on main too cause it's now `0.5.1dev1`

Reply:
Good catch. Push my test code by mistake. Will rebase on main.